File: backend/components/metiers/threed/tests_centralises/legacy/samples/scripts/test_script_sample.py
"""
Test ultra avancé pour script_sample.py
"""

from threed.legacy.samples.scripts.script_sample import script_sample


def test_script_execution():
    assert script_sample("backup") == "Script exécuté pour la tâche : backup"


# ...autres tests avancés, edge cases, robustesse...

File: backend/components/metiers/threed/tests_centralises/legacy/samples/scripts/test_script_sample.js
// Test ultra avancé pour script_sample.js
const scriptSample = require('../../../legacy/samples/scripts/script_sample');

describe('Script Sample', () => {
  it('doit exécuter le script pour une tâche', () => {
    expect(scriptSample('backup')).toMatch('Script exécuté pour la tâche : backup');
  });
  // ...autres tests avancés, edge cases, robustesse...
});
